.ci/check

```diff
@@ -28,8 +28,23 @@ if [[ "${SOURCE_PATH}" != *"src/github.com/gardener/gardener" ]]; then
   export PATH="${GOBIN}:${PATH}"
 fi
 
-# Install Golint (linting tool).
-go get -u golang.org/x/lint/golint
+# Install Golint.
+if ! which golint 1>/dev/null; then
+  echo -n "Installing golint... "
+  go get -u golang.org/x/lint/golint
+  echo "done."
+fi
+
+# Install Helm (see https://docs.helm.sh/using_helm/#from-script).
+if ! which helm 1>/dev/null; then
+  echo -n "Installing helm... "
+  install_helm_path="./get_helm.sh"
+  curl https://raw.githubusercontent.com/helm/helm/master/scripts/get > "${install_helm_path}"
```

Comment:
Changes in the master branch may break our builds.
```suggestion
  curl https://raw.githubusercontent.com/helm/helm/v2.9.1/scripts/get > "${install_helm_path}"
```

Reply:
Nice catch, thanks, will do.